flujo/infra/console_tracer.py

```
@@ -67,12 +67,46 @@ def _handle_pre_run(self, payload: PreRunPayload) -> None:
-        title = "Pipeline End"
-        is_success = all(s.success for s in pipeline_result.step_history)
-        status_text = "✅ COMPLETED" if is_success else "❌ FAILED"
-        status_style = "bold green" if is_success else "bold red"
+
+        # Determine final status
+        paused = False
+        try:
+            ctx = payload.context or getattr(pipeline_result, "final_pipeline_context", None)
+            scratch = getattr(ctx, "scratchpad", None) if ctx is not None else None
+            paused = isinstance(scratch, dict) and scratch.get("status") == "paused"
+        except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)